src/components/Branding/styled-components.js

```diff
@@ -0,0 +1,22 @@
+import styled from 'styled-components';
+import { color } from '../common-styled-components';
+
+export const BrandingContainer = styled.div`
+  position: fixed;
+  top: 0px;
+  width: inherit;
```

Comment:
Change to width: 100% unless you're sure inherit is the way to go

Reply:
Changed to 100%, but also changed position:fixed to absolute.